polkadot/primitives/src/v6/mod.rs

```diff
@@ -393,7 +393,7 @@ pub const MAX_POV_SIZE: u32 = 5 * 1024 * 1024;
 /// Default queue size we use for the on-demand order book.
 ///
 /// Can be adjusted in configuration.
-pub const ON_DEMAND_DEFAULT_QUEUE_MAX_SIZE: u32 = 10_000;
+pub const ON_DEMAND_DEFAULT_QUEUE_MAX_SIZE: u32 = 500;
```

Comment:
Is this sufficient to not build overweight blocks ?

Reply:
This changes will not fix anything. This variable is only used for migrations or at genesis. If you want to lower this, you need to set this in the `HostConfiguration` on chain.